tests/beaker_tests/cisco_bgp/test_bgp_defaults.rb

```diff
@@ -135,6 +135,12 @@
   # Default VRF Test Cases
   # ----------------------
 
+  # delete properties not support on platform
+  device = platform
```

Comment:
@glennmatthews Agreed, out of scope here. I can make the change to re-name the `platform` api to `product_id` and fix up any tests using it.